h5hut/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Any particular reason this requires cmake 3.14? That's rather new. Bionic is only on 3.10 and I don't think telling people to upgrade their cmake (if we say Bionic is supported) is nice -  people will be building other stuff on their system that they don't want to break.
FWIW: it works with 3.13 for me...

Reply:
I think this can be relaxed a bit now, it needed 3.14 to pull in all the SWIG configuration correctly to build the Python module. Using `h5py` means this isn't required. It'd be nice to bump down the version as far as possible (I think the `list(TRANSFORM...` in the Fortran CMakeLists.txt is the newest feature...